src/System.Private.ServiceModel/tools/scripts/WCFCustomSetup.cmd

```diff
@@ -0,0 +1,34 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+REM
+Echo Set Up test services
+REM
+
+REM ServiceUri is set, then the service is not local hosted
+if NOT "%ServiceUri%"=="" (
```

Comment:
If you remove the check, how will this code know it should exit due to not running as localhost?  If you check that condition from another place and only call this when $(ServiceUri) is empty, then this should be renamed WcfSetupSelfHostedWcfService.cmd or something like that.  WcfCustomSetup doesn't describe what it does.  The rename is not blocking but worth considering.